hw_1.2/hm_1.2/Program.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.Text;
+
+namespace hm_1._2
+{
+    class Program
+    {
+        //calculate BWT
+        static string GetBWT(string str)
+        {
+            byte[] strInASCII = Encoding.ASCII.GetBytes(str);
+            int[,] circlePermutations = new int[strInASCII.Length, strInASCII.Length];
```

Comment:
```suggestion
            var circlePermutations = new int[strInASCII.Length, strInASCII.Length];
```
В данном случае тип очевиден из правой части.